frontend/src/components/VolunteerRequest.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Card from '../layouts/CardLayout';
+import CardBody from './CardBody';
+import CardHeader from './CardHeader';
+import i18n from '../common/config/i18n';
+import FormReadOnlyElement from './FormReadOnlyElement';
+import { formatDate } from '../common/utils/utils';
+
+interface VolunteerRequestProps {
+  email: string;
+  phone: string;
+  createdOn: Date;
+  answers: string[];
+}
+
+const VolunteerRequest = ({ email, phone, createdOn, answers }: VolunteerRequestProps) => {
+  return (
+    <Card>
+      <CardHeader>
+        <h2>{i18n.t('registration:form')}</h2>
+      </CardHeader>
+      <CardBody>
+        <div className="flex flex-col divide-y divide-cool-gray-200 gap-6 pb-4">
+          <div className="grid grid-cols-2 gap-8">
+            <FormReadOnlyElement
+              value={email}
+              valueStyle="text-cool-gray-900"
```

Comment:
If all paragraphs, small, labels throughout the app should have this color we should add it in the theme, not pass it to this component, this component should have the same style everywhere. Please validate with @AndreeaMariaP

Reply:
@OlteanuCosmin please remove valueStyle and labelStyle, there is no need for them. Thank you!